src/frontend/src/pages/PIAForm/index.tsx

```diff
@@ -110,16 +117,32 @@ const PIAFormPage = () => {
     setComments(commentArr);
   }, [id, selectedSection]);
 
+  /**
+   * Async callback for getting commentCount within a useEffect hook
+   */
+  const getCommentCount = useCallback(async () => {
```

Comment:
we may need to get away with getting comments at the parent level?

Reply:
we do not need get comments in parent level, so i think we can remove these code